### PR TITLE
Add credentials example in the environment block

### PR DESCRIPTION
**Pull request by gvre**

### CODE REVIEW

File: global-variables.asciidoc [hunk 1]
```diff
@@ -15,7 +15,9 @@ pipeline {
     agent any
     
     environment {
-        FOO="foo"
+        FOO = "foo"
+        AWS_ACCESS_KEY_ID     = credentials('jenkins_aws_secret_key_id')
```
Comment (gsaslis) [inline global-variables.asciidoc]: TIL! 
didn't know this syntax... 

what credential type does it need? secret text?

**Review by gsaslis — APPROVED** [review]
👍

File: global-variables.asciidoc (hunk 1 above)
Comment (gvre) [reply global-variables.asciidoc]: > Jenkins' declarative Pipeline syntax has the credentials() helper method (used within the environment directive) which supports secret text, username and password, as well as secret file credentials

Source: https://jenkins.io/doc/book/pipeline/jenkinsfile/

Comment (gsaslis) [reply global-variables.asciidoc]: 🎉

**Pull request merged**